images/ubuntu2004/scripts/postinstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+echo "ubuntu" | sudo -S cp /etc/apt/sources.list /etc/apt/sources.list.bak
```

Comment:
nit: I know the docs say to do this, but it's so folks running this don't shoot themselves in the foot. We don't need a backup file in a captured image

Reply:
Thanks, removed